crates/sui/src/ptb/ptb_parser/command_token.rs

```diff
@@ -169,6 +169,23 @@ impl CommandToken {
             },
         }
     }
+    pub fn all() -> Vec<&'static str> {
```

Comment:
does it need to be a function? And if it does, can it be a free function?

Reply:
This doesn't need to exist at all anymore I think? Let me see...